src/js/videojs-dash.js

```diff
@@ -70,6 +70,71 @@ class Html5DashJS {
     // element to bind to.
     this.mediaPlayer_.initialize();
 
+    // Retrigger a dash.js-specific error event as a generic HTML5 one
```

Comment:
Though it is more standard, maybe just say "as a player error"

Reply:
Good suggestion, thanks